### PR TITLE
bump go version

### LINKED ISSUE #74: Go version is updated to 1.15
AC:
- [x] Go version is updated to 1.15

### PR DESCRIPTION
**Pull request by BradleyBoutcher**
### What does this PR do?
Bump go version to 1.15

### What ticket does this PR close?
Resolves #74 

#### Change log
- [X] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [X] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [X] This PR does not require updating any documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [0.5.3] - 2019-02-06
 ### Changed
+- The Go version for this project has been updated to `1.15`
```
Comment (diverdane) [inline CHANGELOG.md]: Do we need a CHANGELOG entry for this, i.e.is the Go version a user-facing change for summon-conjur?

**Review by diverdane — COMMENTED** [review]
Looks good. There's a Golang version that needs to be updated in `Dockerfile.test`, and I have a question on the changelog entry.

File: CHANGELOG.md (hunk 1 above)
Comment (BradleyBoutcher) [reply CHANGELOG.md]: I'll strike it

**Review by diverdane — COMMENTED** [review]
Does this line in the `Dockerfile.test` need to be updated?
```
FROM golang:1.13-alpine
```

**Review by diverdane — APPROVED** [review]
LGTM!

**Pull request merged**